docs/zh/UI/quickStart/button.md

```
@@ -0,0 +1,44 @@
+label: UI
+---
+
+`Button` 可以在 UICanvas 中构建交互按钮。
+
+## 编辑器使用
+
+### 添加 Button 节点
+
+在 **[层级面板](/docs/interface/hierarchy/)** 添加 `Button` 节点
+
+<img src="https://mdn.alipayobjects.com/huamei_yo47yq/afts/img/A*Zjn4QbY0B-IAAAAAAAAAAAAAehuCAQ/original" style="zoom:50%;" />
```

Markdownlint (0.37.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)